app/components/account-wrapper/component.js

```diff
@@ -1,5 +1,11 @@
 import Ember from 'ember';
+import service from 'ember-service/inject';
+import { and, not } from 'ember-computed';
 
 export default Ember.Component.extend({
-  classNames: ['account-screen'],
+  audio:             service(),
+  noErrors:          not('audio.hasErrors'),
+  showPlayer:        and('noErrors', 'audio.playedOnce'),
```

Comment:
Would really prefer to pass showPlayer into the account-template component from above, but can't find a good way to get it here from the application template/controller.